gadget/gadget.go

```diff
@@ -44,6 +44,8 @@ const (
 	MBR = "mbr"
 	// GPT identifies a GUID Partition Table partitioning schema
 	GPT = "gpt"
+	// Hybrid identifies a hybrid MBR/GPT schema
+	Hybrid = "mbr,gpt"
```

Comment:
Silly question, is the "mbr,gpt" order fixed? Can it ever be "gpt,mbr"?

Reply:
The docs here https://snapcraft.io/docs/gadget-snap list `mbr,gpt`. I would say it's fixed, unless someone comes in complaining.